src/components/modal/addtostudio/modal.scss

```diff
@@ -0,0 +1,193 @@
+@import "../../../colors";
+@import "../../../frameless";
+
+.mod-addToStudio * {
+    box-sizing: border-box;
+}
+
+.mod-addToStudio {
+    margin: 100px auto;
+    outline: none;
+    padding: 0;
+    width: 36.25rem; /* 580px; */
+    height: 388px; /* 24.25rem; */
```

Comment:
Try to use `rem`s everywhere. In this codebase things are inconsistent, but we should try to be consistent going forward.

Reply:
This is a deliberate decision (per a previous conversation of ours) because of the concern that the bottom of an absolutely positioned element may be impossible to access on small screens with the browser zoomed in.